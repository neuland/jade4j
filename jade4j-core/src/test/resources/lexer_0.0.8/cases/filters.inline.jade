p before #[:cdata inside] after