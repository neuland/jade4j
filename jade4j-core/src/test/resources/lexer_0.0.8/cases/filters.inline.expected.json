{"type":"tag","line":1,"val":"p","selfClosing":false}
{"type":"text","line":1,"val":"before "}
{"type":"start-jade-interpolation","line":1}
{"type":"filter","line":1,"val":"cdata"}
{"type":"text","line":1,"val":"inside"}
{"type":"end-jade-interpolation","line":1}
{"type":"text","line":1,"val":" after"}
{"type":"eos","line":1}